CMakeLists.txt

```diff
@@ -237,7 +236,7 @@ add_definitions(-DBOOST_TEST_DYN_LINK)
 # This way we can skip calls to functions defined in omp.h with code like:
 # if (HAS_OPENMP == 1) { openMP code here }
 # If OpenMP is found, define HAS_OPENMP to be 1. Otherwise define it to be 0.
-find_package(OpenMP 3)
+find_package(OpenMP 3.0.0 )
```

Comment:
This weird thing is happening in AppVeyor:
In line 283, it performs the OpenMP test and actually does find version 3.0.0:
```
277 -- Try OpenMP C flag = [/openmp]
278 -- Performing Test OpenMP_FLAG_DETECTED
279 -- Performing Test OpenMP_FLAG_DETECTED - Success
280 -- Try OpenMP CXX flag = [/openmp]
281 -- Performing Test OpenMP_FLAG_DETECTED
282 -- Performing Test OpenMP_FLAG_DETECTED - Success
283 -- Found OpenMP: /openmp (Required is at least version "3.0.0") 
```
Shouldn't the build actually not find OpenMP since Visual Studio only supports version 2.0 [according to this?](https://msdn.microsoft.com/en-us/library/tt15eb9t.aspx)

This is the cause for the failure - My CMake code thinks it found OpenMP > 3, but it hasn't...